code/java/delete1.java

```diff
@@ -0,0 +1,34 @@
+package code.java;
+
+import org.xapian.WritableDatabase;
+import org.xapian.XapianConstants;
+
+public class delete1 {
+    
+    // Command line args - dbpath identifiers...
+    public static void main(String[] args)
+    {
+        if(args.length < 2)
+        {
+            System.out.println("Insufficient number of arguments (should be dbpath identifiers...)");
+            return;
+        }
+        deleteDocs(args[0], args);
+    }
+    
+    public static void deleteDocs(String dbpath, String[] identifierArgs)
```

Comment:
This is strictly incorrect, because you aren't slicing the array within `main()`.